Library/Formula/clang-format.rb

```diff
@@ -1,15 +1,13 @@
 class ClangFormat < Formula
   desc "Formatting tools for C, C++, Obj-C, Java, JavaScript, TypeScript"
   homepage "http://clang.llvm.org/docs/ClangFormat.html"
-  version "2016-03-08"
+  version "2016-03-29"
 
   stable do
-    url "http://llvm.org/releases/3.8.0/llvm-3.8.0.src.tar.xz"
-    sha256 "555b028e9ee0f6445ff8f949ea10e9cd8be0d084840e21fbbe1d31d51fc06e46"
+    url "http://llvm.org/svn/llvm-project/llvm/tags/google/testing/2016-03-29/", :using => :svn
```

Comment:
The `clang-format` formula was always using the tagged Google releases, the full LLVM releases are too infrequent (often several months apart). Somebody changed that in 20a0a92b39851b60eac8e03338585605e615c758, this just reverts back to the higher frequency releases.